cmd/test/main.go

```diff
@@ -194,20 +207,24 @@ func New(ctx *TestCommandContext) *cobra.Command {
 			return nil
 		},
 	}
+	testCommandFlags.AddFlags(testCommand)
+	return testCommand
+}
 
+// AddFlags registers flags for a cli
+func (flags *TestCommandFlags) AddFlags(cmd *cobra.Command) {
 	if ciContext.Extract() != nil && ciContext.Extract().CIMetadata.ShouldHideEmojis {
-		testCommand.Flags().StringP("output", "o", "simple", "Define output format")
+		cmd.Flags().StringVarP(&flags.Output, "output", "o", "simple", "Define output format")
```

Comment:
Great! I suggest moving this logic into a dedicated method of `TestCommandFlags`(maybe name it `AddFlags`?). If you have any questions about how to do it, don't hesitate to reach out 👍🏼

Reply:
Could you please elaborate more about the above because I have already added flag logic inside `AddFlags` Or do you want that `output` flag logic in another function and call inside `AddFlags`?

Reply:
Oh, I'm terribly sorry, I misread it. Great job!